docs/source/reference/federation/errors.mdx

```diff
@@ -34,7 +34,91 @@ The following errors might be raised during composition:
 <tr>
 <td>
 
-##### `DEFAULT_VALUE_USES_INACCESSIBLE`
+###### `ABSOLUTE_CONNECT_URL_WITH_SOURCE`
+
+Since v2.10.0
+
+</td>
+<td>
+
+The `@connect` is using a `source`, but the URL is absolute. This is not allowed because the `@source` URL is joined with the `@connect` URL, so the `@connect` URL should actually be a path only.
```

Comment:
I've added slight copyedits to some descriptions and in some cases added them. (I've flagged when I'm not sure the description I've added is accurate.)